tests/routes/post_stock_test.py

```diff
@@ -1,260 +1,275 @@
-
-        @patch('pcapi.routes.stocks.redis.add_offer_id')
-        @pytest.mark.usefixtures("db_session")
-        def when_stock_is_created_expect_offer_id_to_be_added_to_redis(self, mock_add_offer_id_to_redis, app):
-            # Given
-            user = create_user()
-            offerer = create_offerer()
-            create_user_offerer(user, offerer)
-            venue = create_venue(offerer)
-            offer = create_offer_with_thing_product(venue)
-            repository.save(offer)
-            stock_data = {'price': 1222, 'offerId': humanize(offer.id)}
-
-            # When
-            TestClient(app.test_client()) \
-                .with_auth(user.email) \
-                .post('/stocks', json=stock_data)
-
-            # Then
-            mock_add_offer_id_to_redis.assert_called()
-            mock_args, mock_kwargs = mock_add_offer_id_to_redis.call_args
-            assert mock_kwargs['offer_id'] == offer.id
-
-    class Returns400:
-        @pytest.mark.usefixtures("db_session")
-        def when_missing_offer_id(self, app):
-            # Given
-            user = create_user(can_book_free_offers=False, email='test@email.fr', is_admin=True)
-            offerer = create_offerer()
-            venue = create_venue(offerer)
-            offer = create_offer_with_thing_product(venue)
-            repository.save(user, offer)
-
-            # When
-            response = TestClient(app.test_client()).with_auth(user.email) \
-                .post('/stocks', json={'price': 1222})
-
-            # Then
-            assert response.status_code == 400
-            assert response.json["offerId"] == ['Ce paramètre est obligatoire']
-
-        @pytest.mark.usefixtures("db_session")
-        def when_booking_limit_datetime_after_beginning_datetime(self, app):
-            # Given
-            user = create_user(can_book_free_offers=False, email='email@test.com', is_admin=True)
-            offerer = create_offerer()
-            venue = create_venue(offerer)
-            offer = create_offer_with_event_product(venue)
-            repository.save(user, offer)
-
-            beginningDatetime = datetime(2019, 2, 14)
-
-            data = {
-                'price': 1222,
-                'offerId': humanize(offer.id),
-                'beginningDatetime': serialize(beginningDatetime),
-                'bookingLimitDatetime': serialize(beginningDatetime + timedelta(days=2))
-            }
-
-            # When
-            response = TestClient(app.test_client()).with_auth(user.email) \
-                .post('/stocks', json=data)
-
-            # Then
-            assert response.status_code == 400
-            assert response.json['bookingLimitDatetime'] == [
-                'La date limite de réservation pour cette offre est postérieure à la date de début de l\'évènement'
+from pcapi.utils.date import DATE_ISO_FORMAT
+
+
+class Returns201:
+    @patch("pcapi.routes.mediations.feature_queries.is_active", return_value=True)
```

Comment:
```suggestion
    @override_features(SYNC_ALGOLIA=True)
```

(importé via `from pcapi.models.feature import override_features`). J'ai ajouté ce [décorateur `override_features()`](https://github.com/pass-culture/pass-culture-api/blob/9548395f2011d031b1680ed5df395e13ff54ba6b/src/pcapi/models/feature.py#L47-L64) qui me semble plus lisible (et plus précis sur la feature qu'on active ou désactive).